### PR TITLE
Added new exception page

### PR DESCRIPTION
**Pull request by mamantoha**
This PR adds a new exception page for 500 error in the development environment, and use [exception_page](https://github.com/crystal-loot/exception_page) shard.

Production environment still use basic error page.

### CODE REVIEW

**Comment by mamantoha** [comment]
exception_page

File: src/kemal/exception_page.cr [hunk 1]
```diff
@@ -0,0 +1,33 @@
+require "exception_page"
+
+module Kemal
+  class ExceptionPage < ExceptionPage
+    def styles
+      ExceptionPage::Styles.new(
+        accent: "purple"
```
Comment (paulcsmith) [inline src/kemal/exception_page.cr]: Maybe the Kemal red would look good here?

**Comment by mamantoha** [comment]
Hi @paulcsmith .

This PR is mostly about the ability to include this in Kemal. After approval, the styles should be changed, of course.

**Comment by mamantoha** [comment]
exception_page01

**Comment by mamantoha** [comment]
Hi @sdogruyol .

For me, it looks very superior.
Feel free to merge this as is. Or create another PR based on this code.
Thanks!

**Comment by sdogruyol** [comment]
Wow, this looks amazing ❤️ Thanks a lot @mamantoha 👍

**Pull request merged**